ui/tests/unit/serializers/pki/config-test.js

```diff
@@ -0,0 +1,23 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'vault/tests/helpers';
+
+module('Unit | Serializer | pki/config', function (hooks) {
+  setupTest(hooks);
+
+  // Replace this with your real tests.
```

Comment:
same question here!

Reply:
Leaving this one as-is so we can write tests when the serializer does stuff